lib/nimble_template/addons/variants/phoenix/web/svg_sprite.ex

```diff
@@ -5,115 +5,109 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.SvgSprite do
-  #         "svg-sprite.generate-icon": "svg-sprite --shape-id-generator \\"icon-%s\\" --symbol --symbol-dest ../priv/static/images --symbol-sprite icon-sprite.svg ../priv/static/images/icons/*.svg",
-  #     """
-  #   )
-
-  #   Generator.replace_content!(
-  #     "assets/package.json",
-  #     """
-  #       "devDependencies": {
-  #     """,
-  #     """
-  #       "devDependencies": {
-  #         "svg-sprite": "1.5.4",
-  #     """
-  #   )
-
-  #   project
-  # end
-
-  # defp edit_web_entry_point!(%Project{web_module: web_module, web_path: web_path} = project) do
-  #   Generator.replace_content!(
-  #     "#{web_path}.ex",
-  #     """
-  #           # HTML escaping functionality
-  #           import Phoenix.HTML
-  #     """,
-  #     """
-  #           # HTML escaping functionality
-  #           import Phoenix.HTML
-
-  #           import #{web_module}.IconHelper
-  #     """
-  #   )
-
-  #   project
-  # end
-
-  # defp edit_wiki_sidebar!(project) do
-  #   if has_github_wiki_directory?() do
-  #     Generator.replace_content!(
-  #       ".github/wiki/_Sidebar.md",
-  #       """
-  #       - [[Getting Started]]
-  #       """,
-  #       """
-  #       - [[Getting Started]]
-
-  #       ## Operations
-
-  #       - [[Icon Sprite]]
-  #       """
-  #     )
-  #   end
-
-  #   project
-  # end
-
-  # defp copy_icon_helper!(
-  #        %Project{web_module: web_module, web_path: web_path, web_test_path: web_test_path} =
-  #          project
-  #      ) do
-  #   Generator.copy_file!(
-  #     [
-  #       {:eex, "lib/otp_app_web/helpers/icon_helper.ex.eex", "#{web_path}/helpers/icon_helper.ex"},
-  #       {:eex, "test/otp_app_web/helpers/icon_helper_test.exs.eex",
-  #        "#{web_test_path}/helpers/icon_helper_test.exs"}
-  #     ],
-  #     web_module: web_module
-  #   )
-
-  #   project
-  # end
-
-  # defp copy_wiki_documentation!(%Project{} = project) do
-  #   if has_github_wiki_directory?() do
-  #     Generator.copy_file!([{:text, ".github/wiki/Icon-Sprite.md", ".github/wiki/Icon-Sprite.md"}])
-  #   end
-
-  #   project
-  # end
+  defp edit_files!(%Project{} = project) do
+    project
+    |> edit_assets_package!()
+    |> edit_web_entry_point!()
+    |> edit_wiki_sidebar!()
+  end
+
+  defp copy_files!(%Project{} = project) do
+    project
+    |> copy_icon_helper!()
+    |> copy_wiki_documentation!()
+  end
+
+  defp edit_assets_package!(%Project{} = project) do
+    Generator.replace_content!(
+      "assets/package.json",
+      """
+        "scripts": {
+      """,
+      """
+        "scripts": {
+          "svg-sprite.generate-icon": "svg-sprite --shape-id-generator \\"icon-%s\\" --symbol --symbol-dest ../priv/static/images --symbol-sprite icon-sprite.svg ../priv/static/images/icons/*.svg",
+      """
+    )
+
+    Generator.replace_content!(
+      "assets/package.json",
+      """
+        "devDependencies": {
+      """,
+      """
+        "devDependencies": {
+          "svg-sprite": "1.5.4",
```

Comment:
can we take this chance to upgrade the `svg-sprite` to `2.0.2` @liamstevens111 :D if no, we can do it on a separate PR ;)

Reply:
nevermind, let's do this as a separate task, I will mark this as resolved